common/lib/src/realm_common_base.dart

```diff
@@ -97,3 +97,11 @@ class Indexed {
 class Ignored {
   const Ignored();
 }
+
+/// Indicates that the field it decorates is the inverse end of a relationship.
+/// {@category Annotations}
+class Backlink {
+  /// The name of the field in the other class that links to this class.
```

Comment:
.NET has this as a doc `The property that is on the other end of the relationship.` Should we use
```suggestion
  /// The name of the field that is on the other end of the relationship.
```

Reply:
I kind of think my sentence is a bit more precise? Also the suggestion is almost identical to the comment in line 101